subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_df.c

```diff
@@ -26,46 +28,19 @@
  */
 #define DF_PSEL_GPIO_NUM_MAX_IDX 7
 
-#if IS_ENABLED(CONFIG_BT_CTLR_DF_INIT_ANT_SEL_GPIOS)
-struct dfe_gpio_psel {
```

Comment:
This PR adds this structure in the previous commit, only to remove it immediately. Can you please clean up the commits so this is easier to review and doesn't add and remove things like that? It looks like some of this commit should have been squashed into the previous commit instead during a rebase.

Reply:
I've removed the code from previous commit.